config/v1/types_proxy.go

```diff
@@ -34,6 +34,10 @@ type ProxySpec struct {
 	// Empty means unset and will not result in an env var.
 	// +optional
 	NoProxy string `json:"noProxy,omitempty"`
+
+	// readinessEndpoints is a list of endpoints used to verify readiness of the proxy.
+	// +optional
+	ReadinessEndpoints *[]string `json:"readinessEndpoints,omitempty"`
```

Comment:
i don't think we typically use pointers to slices in our api.  just make it a slice.

Reply:
@bparees so I understand correctly, the installer will then need to create an empty slice, correct?

Reply:
no, it can just leave the field missing.

Reply:
@bparees I updated `ReadinessEndpoints` to be a string.